src/android/patches/0.20/addPackagePatch.js

```diff
@@ -1,2 +0,0 @@
-module.exports = (config) =>
```

Comment:
We don't have an addPackagePatch anymore. We use importPatch and packagePatch instead.